### DISCUSSION TITLE
"AttributeError:  'StationaryModel' object has no attribute 'parameters'" in the "Getting started" example

### DISCUSSION THREAD

**Discussion by cyu2020** [pymor/pymor#1353] [Q&A]
Thanks for sharing this amazing PyMOR repository.

I was trying out PyMOR with the "Getting started" example following the documentation. When trying to "print(fom.parameters)", I got the "AttributeError:  'StationaryModel' object has no attribute 'parameters'". This would also be a problem when trying to build the "reductor". But it would work fine if I remote the "coercivity_estimator" argument in the "CoerciveRBReductor" function. I am not sure if this is a PyMOR issue or I did not set up PyMOR correctly (I installed PyMOR with anaconda).

Thanks,

CY

**Comment by renefritze** [comment]
Hey @cyu2020! Thank you for reaching out with your problem. This sounds like a mismatch of the installed version of pyMOR and the documentation you're trying to follow. You can confirm which version is installed with `python -c "import pymor; print(pymor.__version__)"`
You probably have 2019.2.0 or earlier installed. Compare the old ["Getting Started"](https://docs.pymor.org/2019.2.0/getting_started.html)

If you installed via the [conda-forge channel](https://anaconda.org/conda-forge/pymor), you _should_ have 2020.2.0

**Reply by cyu2020** [reply]
I tried "print(pymor.__version__)", but it printed out "0+unknown". I also tried "print(fom.parameter_type)" from the old "Getting started" documentation and it works fine. So it seems to be an old version. But I did install via "conda install -c conda-forge pymor".

**Reply by renefritze** [reply]
The version bit is definitely an [issue](https://github.com/pymor/pymor/issues/1354) we'll look into. 

As for the outdated install itself, I'm stumped. Is your platform maybe not one of `win-64, osx-64, linux-64`? Which python version is used in your conda setup?

**Comment by cyu2020** [comment]
My platform is win-64. My python version is 3.7.3 in conda of version 4.10.3
